pages/articles/templates/articles/index.html

```diff
@@ -0,0 +1,192 @@
+{% extends "base.html" %}
+{% load static %}
+
+{% block content %}
+<!-- Main Content -->
+<div class="py-8" id="content">
+    <!-- Page Header -->
+    <div class="mb-8">
+        <p>
+            Two types of articles (data highlights and editorials) are generated by the portal and its community. 
+            Data highlights showcase recent Swedish publications sharing data, whilst editorials are short opinion-style 
+            pieces discussing the current thinking in a given area.</p>
+        <p>
+            Use the tags or search bar to find data highlights or editorials of potential interest. 
+            If you are interested in having your work featured, please <a href="/contact/">contact us</a>.
+        </p>
+    </div>
+    
+    <!-- Search Bar -->
+    <div class="mb-8 flex justify-end">
+        <input type="text" 
+               id="searchInput" 
+               class="w-80 px-4 py-2 border border-pp-pale-grey rounded-lg 
+                      focus:ring-2 focus:ring-pp-teal focus:border-pp-teal 
+                      text-pp-dark-grey placeholder-pp-dark-grey bg-white" 
+               placeholder="Search Articles" 
+               aria-label="Search">
+    </div>
+    
+    {% if articles %}
+        <div id="articles_container" 
+             class="grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3 gap-2 lg:gap-3">
+            {% for article in articles %}
+                <div class="data-highlight" 
+                     data-search-text="{{ article.title|lower }} 
+                                      {{ article.summary|lower }} 
+                                      {% for topic in article.topics.all %}
+                                          {{ topic.name|lower }} 
+                                      {% endfor %}">
+                    <div class="h-full px-2.5 py-3 rounded-sm bg-[#d8d8d8] hover:shadow-[0px_0px_3px_#55585a] relative">
```

Comment:
I know we will do the styling later, but just wanna mention it fore hand (will do again later too).

I am not a fan of this being here, we should try different styles later. For example, there is already vacant space beside the date, maybe we can try placing it there. We could play with it when finalising this page styling, not need to be fixed now 👍